### PR TITLE
Redirect all recording links from tests.replay.io to app.replay.io

### PR DESCRIPTION
**Pull request by jaril**

### CODE REVIEW

File: src/ui/utils/recording.ts [hunk 1]
```diff
@@ -15,9 +16,23 @@ export function getRecordingURL(recording: Recording): string {
...
 
+  // Revert once we merge the test-suites branch into main (see SCS-1723)
+  if (window.location.host === "tests.replay.io") {
+    return `https://app.replay.io/recording/${id}`;
+  }
+
   return `/recording/${id}`;
 }
 
+export function useGetRecordingURLForTest(recordingId: string): string {
```
Comment (ryanjduffy) [inline src/ui/utils/recording.ts:27]: Will this mean that users will get "stuck" on app.replay.io once they open a recording? E.g. the in app back button will keep them on app.replay.io

Comment (jasonLaster) [reply src/ui/utils/recording.ts:27]: That's a good point. 

If this is a problem in the future we might need to use a NextJS middleware to redirect requests for the library when you're viewing a test workspace. Hopefully this is temporary though and we dont need it.

Comment (jaril) [reply src/ui/utils/recording.ts:27]: @ryanjduffy still ok to just merge so we stop breaking telemetry?

Comment (jasonLaster) [reply src/ui/utils/recording.ts:27]: also probably good to do since test-suites will not have the latest and greatest

**Review by ryanjduffy — CHANGES_REQUESTED** [review]
I tried this locally and it does behave as I guessed above. If all we're trying to do with this is fix sending telemetry, let's fix that problem instead. We need to be merging `main` into this branch regularly anyway so we shouldn't fall behind in functionality or fixes.

**Pull request closed without merging**